File: Flink/src/main/scala/Bean/BookBean.scala
package Bean

import scala.beans.BeanProperty

/**
 * /**
 * ID,Movie_Name_EN,Movie_Name_CN,Crawl_Date,Number,Username,Date,Star,Comment,Like
 * 0,Avengers Age of Ultron,复仇者联盟2,2017-01-22,1,然潘,2015-05-13,3, 连奥创都知道整容要去韩国。,2404
 * */
 *
 * @param ID
 * @param Movie_Name_EN
 * @param Movie_Name_CN
 * @param Crawl_Date
 * @param Number
 * @param Username
 * @param Date
 * @param star
 * @param Comment
 * @param Like
 */

 case class BookBean( var ID: String ,
                     var Movie_Name_EN: String ,
                     var Movie_Name_CN: String ,
                     var Crawl_Date: String ,
                     var Number: String,
                     var Username: String,
                     var Date:String,
                     var Star:String,
                     var Comment:String,
                     var Like:String
                   ){



}

File: flink-api/src/main/java/com/strategyopr/flink/bean/BookBean.java
package com.strategyopr.flink.bean;

public class BookBean {
     String ID ;
     String Movie_Name_EN ;
    String Movie_Name_CN ;
    String Crawl_Date ;
    String Number;
    String Username;
    String Date;
    String Star;
    String Comment;
    String Like;

    public BookBean(String ID, String movie_Name_EN, String movie_Name_CN, String crawl_Date, String number, String username, String date, String star, String comment, String like) {
        this.ID = ID;
        Movie_Name_EN = movie_Name_EN;
        Movie_Name_CN = movie_Name_CN;
        Crawl_Date = crawl_Date;
        Number = number;
        Username = username;
        Date = date;
        Star = star;
        Comment = comment;
        Like = like;
    }

    public String getID() {
        return ID;
    }

    public void setID(String ID) {
        this.ID = ID;
    }

    public String getMovie_Name_EN() {
        return Movie_Name_EN;
    }

    public void setMovie_Name_EN(String movie_Name_EN) {
        Movie_Name_EN = movie_Name_EN;
    }

    public String getMovie_Name_CN() {
        return Movie_Name_CN;
    }

    public void setMovie_Name_CN(String movie_Name_CN) {
        Movie_Name_CN = movie_Name_CN;
    }

    public String getCrawl_Date() {
        return Crawl_Date;
    }

    public void setCrawl_Date(String crawl_Date) {
        Crawl_Date = crawl_Date;
    }

    public String getNumber() {
        return Number;
    }

    public void setNumber(String number) {
        Number = number;
    }

    public String getUsername() {
        return Username;
    }

    public void setUsername(String username) {
        Username = username;
    }

    public String getDate() {
        return Date;
    }

    public void setDate(String date) {
        Date = date;
    }

    public String getStar() {
        return Star;
    }

    public void setStar(String star) {
        Star = star;
    }

    public String getComment() {
        return Comment;
    }

    public void setComment(String comment) {
        Comment = comment;
    }

    public String getLike() {
        return Like;
    }

    public void setLike(String like) {
        Like = like;
    }
}
